wiki/Client/Beatmap_editor/Menu/de.md

```diff
@@ -203,9 +198,9 @@ Interessante Dinge.
 | :-- | :-- |
 | Sektion hinzufügen (`Strg` + `P`) | Erstellt eine neue rote Timing-Sektion am derzeitigen Standort (neue BPM). |
```

Comment:
I feel like changing this to

```suggestion
| Sektion hinzufügen (`Strg` + `P`) | Erstellt eine neue rote Timing-Sektion an der derzeitigen Position (neue BPM). |
```

It could also be removed because the English text only mentions a red offset. The same applies for the line below.

Reply:
Probably fine to just remove. 🤔

Reply:
```suggestion
| Sektion hinzufügen (`Strg` + `P`) | Erstellt eine neue rote Timing-Sektion (neue BPM). |
```